mocha/index.js

```diff
@@ -5,8 +5,16 @@ module.exports = {
     "mocha": true
   },
 
+  "plugins": [
+    "mocha"
+  ],
+
   "rules": {
     // Deeply nested functions are a reality of Mocha+BDD, so permit them
-    "max-nested-callbacks": 0
+    "max-nested-callbacks": 0,
+    "mocha/handle-done-callback": 2,
+    "mocha/no-exclusive-tests": 1,
```

Comment:
Should this be a warning instead of an error? It could easily get past us on CI otherwise.